docs/source/usage/torch_compile.rst

```diff
@@ -0,0 +1,200 @@
+.. _torch_compile:
+
+Tuning ``torch.compile``
+========================
+
+vLLM provides several options for users to optimize the performance of their models using ``torch.compile``. As the integration progresses, more flags will become available. Currently, there are two main flags that users can utilize:
+
+- ``level``: Set this to 3 for optimal performance. Other levels are intended for internal testing and debugging.
+- ``candidate_compile_sizes``: A list of batch sizes for which the user wants to compile the model. Only the batch sizes that are part of cudagraph capture sizes will be compiled.
+
+To effectively use ``torch.compile``, the TL;DR; is:
+
+- Ensure GPUs are busy executing the model before enabling ``torch.compile``.
```

Comment:
Can you clarify this statement?  I think I know what it is trying to say but I think it is a bit ambiguous.

Reply:
do you have better ideas?

Reply:
Maybe something like "`torch.compile` works best for models that are GPU bound"?